packages/cf.js/API_REFERENCE.md

```diff
@@ -324,7 +324,15 @@ TODO
 
 ### Method: `rejectState`
 
-TODO
+Rejects a proposed state to an AppInstance and notifies the counterparty of
+said rejection.
+
+  Params:
+- `appInstanceId: string`
```

Comment:
Shouldn't you pass in like, the hash of the state that was proposed, so you can match it up